definitions/pipelines/rnaseq_star_fusion.cwl

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "STAR-RNA-Seq alignment and transcript/gene abundance workflow"
+requirements:
+    - class: MultipleInputFeatureRequirement
+    - class: SubworkflowFeatureRequirement
+    - class: ScatterFeatureRequirement
+    - class: InlineJavascriptRequirement
+inputs:
+    instrument_data_bams:
+        type: File[]
+    outSAMattrRGline:
+        type: string[]
+    stargenomeDir:
+        type: Directory
+    gtf_file:
+        type: File
+    trimming_adapters:
+        type: File
+    trimming_adapter_trim_end:
+        type: string
+    trimming_adapter_min_overlap:
+        type: int
+    trimming_max_uncalled:
+        type: int
+    trimming_min_readlength:
+        type: int
+    kallisto_index:
+       type: File
+    gene_transcript_lookup_table:
+       type: File
+    strand:
+       type: string?
+    refFlat:
+        type: File
+    ribosomal_intervals:
+        type: File
+    sample_name:
+        type: string
+outputs:
+    final_bam:
+        type: File
+        outputSource: index_bam/indexed_bam
+        secondaryFiles: [.bai]
+    stringtie_transcript_gtf:
+        type: File
+        outputSource: stringtie/transcript_gtf
+    stringtie_gene_expression_tsv:
+        type: File
+        outputSource: stringtie/gene_expression_tsv
+    transcript_abundance_tsv:
+        type: File
+        outputSource: kallisto/expression_transcript_table
+    transcript_abundance_h5:
+        type: File
+        outputSource: kallisto/expression_transcript_h5
+    gene_abundance:
+        type: File
+        outputSource: transcript_to_gene/gene_abundance
+    metrics:
+        type: File
+        outputSource: generate_qc_metrics/metrics
+    chart:
+        type: File
+        outputSource: generate_qc_metrics/chart
+    fusion_evidence:
+        type: File
+        outputSource: kallisto/fusion_evidence
+steps:
+    bam_to_trimmed_fastq_and_star_fusion_alignments:
+        run: ../subworkflows/bam_to_trimmed_fastq.cwl
+        scatter: [bam]
+        scatterMethod: dotproduct
+        in:
+            bam: instrument_data_bams
+            adapters: trimming_adapters
+            adapter_trim_end: trimming_adapter_trim_end
+            adapter_min_overlap: trimming_adapter_min_overlap
+            max_uncalled: trimming_max_uncalled
+            min_readlength: trimming_min_readlength
+        out:
+            [fastqs, fastq1, fastq2]
+    star_align_fusion:
+        run: ../tools/star_align_fusion.cwl
+        in:
+            outSAMattrRGline: outSAMattrRGline
+            stargenomeDir: stargenomeDir
+            gtf_file: gtf_file
+            fastq:
+                source: bam_to_trimmed_fastq_and_star_fusion_alignments/fastq1
```

Comment:
These two inputs need to be changed to `bam_to_trimmed_fastq` as well, if you update the above